### ISSUE TITLE
Admin rights

### ISSUE THREAD

**Issue by mattgwwalker** [mattgwwalker/msg-extractor#154]
@TheElementalOfDestruction, thanks for your continued stewardship of this project.

I was working on another project and checked out the settings on this project to see how we'd managed to get two admins setup.  And then I saw that you still don't have admin rights.

For the other project I am working on ([PyOgg](https://github.com/TeamPyOgg/PyOgg)) I worked out how to get two Admin-level users.  I created an Organization within GitHub specifically for the project (TeamPyOgg) and then the project was transferred to that organization.  Organizations however require two Owners.

Would you like me to do the same for msg-extractor?  Would you like to co-own the project with me?  This would give you admin-level rights and the ability to setup things like webhooks for documentation based on Read the Docs, or Continuous Integration (CI) using Travis CI on GitHub.

Let me know what you think.

Cheers,

Matthew

**Comment by TheElementalOfDestruction** [comment]
Sounds good, but if that would cause the url to change then I will need a bit to make sure that all of the links get updated, preferable right before the change if possible.

**Comment by punkrokk** [comment]
Git hub will still resolve the repo backwards compatible.

https://docs.github.com/en/github/administering-a-repository/transferring-a-repository

**Comment by mattgwwalker** [comment]
Yes, from the experience I had with PyOgg, the original address is
immediately forwarded to the new one. So for example a google search result
which returned the old address nonetheless worked as github forwarded the
user on to the new address.

**Comment by TheElementalOfDestruction** [comment]
Okay then, that sounds good

**Comment by mattgwwalker** [comment]
Done.

You should now find that the project is automatically forwarded to https://github.com/TeamMsgExtractor/msg-extractor.

I've invited you as an owner of TeamMsgExtractor, so we now share the ownership and you should have full admin-level access to both the organization and the project.  The easiest way to confirm this is that you should see a "Settings" tab on the far right of the project's GitHub page.

Thanks again for all your work.

Matthew
